dockerfiles/Dockerfile.android-maze-runner

```diff
@@ -1,3 +1,8 @@
 FROM 855461928731.dkr.ecr.us-west-1.amazonaws.com/maze-runner-releases:latest-v3-cli as android-maze-runner
 
-COPY features features
+COPY features/scripts features/scripts
+COPY features/steps features/steps
+COPY features/support features/support
+
+ARG TESTS_DIR=features/full_tests
+COPY $TESTS_DIR $TESTS_DIR
```

Comment:
Is it worth just always copying all the tests, or would there be some side effect to that which is undesirable?

It would definitely be good to run the `_smoke_test.feature` on the full CI run either way - as that should get quick feedback about whether a feature is successful when running against `next/master`.

Reply:
We could have a single maze-runner container and leave the individual config to the step in the pipeline, I'm not opposed to that

Reply:
In the Cocoa pipeline we consume the Maze Runner image directly in `docker-compose.yml` and mount `features/` as a volume.  There is no build step in the pipeline at all - would that be an option here?  Although it takes next to no time to build the image, there is instance wait time and general use to consider.  We could have an annoying wait time to get a fresh agent, just for it to then sit idle for 30 minutes before being surrendered.

Reply:
Yes, that would make a lot more sense